async function createUser(req, res){
    try{

    } catch (e){
        res.status(400).json({
            message: "Error creating user"
        })
    }
}
